### ISSUE TITLE
Push notifications for Android and iOS

### ISSUE THREAD

**Issue by lgeiger** [ekonstantinidis/gitify-mobile#25]
It's awesome that gitify is now available on iOS and Android as well!

It would be great if gitify-mobile would support push notifications like the desktop app.

**Comment by ajkblue** [comment]
Has any progress been made on this?

**Comment by FezVrasta** [comment]
I bought the app thinking that this feature was already included actually.

Is it possible to at least add the notifications counter (the red badge) on the app icon?

**Comment by fvdm** [comment]
I agree with @FezVrasta, to me it feels a bit misleading. I can view notifications on Github.com, don't need an app for that. For a mobile app that is only meant for handling notifications I expected push notification support.

I'm not angry, just disappointed. I'm happy to pay for a subscription if that is required to help cover the cost.

**Comment by nlundquist** [comment]
I also agree with @FezVrasta it's misleading that the app doesn't already support this. 

Disappointing there hasn't been a comment regarding this issue from the developer yet.
